README.md

```diff
@@ -362,6 +362,20 @@ Make sure lockfile version is higher than v1 (Node.js 15 or higher).
 
 See [docs/npm.md](docs/npm.md) for more details.
 
+### yarn
+
+<https://v3.yarnpkg.com/>
+
+Current version: v3
+
+Cachi2 supports Yarn by parsing [package.json](https://v3.yarnpkg.com/configuration/manifest/)
+file present in the source repository and downloading the declared dependencies.
+
+If not checked in to the repository, a lockfile (``yarn.lock``) will automatically be generated by
```

Comment:
```suggestion
If not already checked in to the repository, a lockfile (``yarn.lock``) will automatically be generated by
```

Reply:
I'll drop this sentence with https://github.com/containerbuildsystem/cachi2/pull/425#discussion_r1426765531 in mind.